city_scrapers/spiders/wayne_local_emergency_planning.py

```diff
@@ -0,0 +1,99 @@
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil import parser
+
+
+class WayneLocalEmergencyPlanningSpider(CityScrapersSpider):
+    name = "wayne_local_emergency_planning"
+    agency = "Wayne County Local Emergency Planning Committee"
+    timezone = "America/Detroit"
+    allowed_domains = ["www.waynecounty.com"]
+    start_urls = ["https://www.waynecounty.com/departments/hsem/wayne-county-lepc.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        # the meeting dates I've seen all had the substring 'day, ' as in 'Wednesday, March 6, 2019'
+        meeting_dates = response.xpath('''//p[contains(text(),'day, ')]''')
+
+        for item in meeting_dates:
+            # clean off the paragraph tags from the parsed meeting date (item)
+            item = str(item.extract()).replace('<p>', '').replace('</p>', '')
+            # print('------------')
```

Comment:
No worries about these as placeholders while you're working on it, but just leaving this as a reminder to remove them before it's merged